scss/_base_syntax-highlighting.scss

```diff
@@ -13,11 +13,11 @@
     &.prolog,
     &.doctype,
     &.cdata {
-      color: $colors--light-theme--text-muted;
+      color: $colors--theme--text-muted;
```

Comment:
We probably need to look into theming rest of the syntax highlighting colours, but this can wait.